src/java/org/apache/cassandra/db/partitions/AbstractBTreePartition.java

```diff
@@ -108,10 +112,29 @@ public EncodingStats stats()
 
     public Row getRow(Clustering clustering)
     {
-        Row row = searchIterator(ColumnFilter.selection(columns()), false).next(clustering);
-        // Note that for statics, this will never return null, this will return an empty row. However,
```

Comment:
good idea, updated the javadoc

Reply:
Nice, though I meant adding that JavaDoc and tag to it's super method `Partition#getRow`, since I think it applies to all implementations at the moment, but I might be wrong. WDYT?

Reply:
moved to `Partition#getRow`, didn't notice the interface..